packages/mds-compliance-engine/engine/helpers.ts

```diff
@@ -151,7 +151,7 @@ export function isRuleActive({ start_time, end_time, days }: Pick<Rule, 'start_t
   return isCurrentDayInDays({ days }) && isCurrentTimeInInterval({ start_time, end_time })
 }
 
-export function isInVehicleTypes(rule: Rule, device: Device): boolean {
+export function isInVehicleTypes(rule: Rule, device: Pick<Device, 'vehicle_type'>): boolean {
```

Comment:
If you wanna be reeeeeallly fancy, you could destructure the `vehicle_type` property from the input param:
```suggestion
export function isInVehicleTypes(rule: Rule, { vehicle_type }: Pick<Device, 'vehicle_type'>): boolean {
```

(note, would have to change the usage a tad in the function too, but I don't think I can add a github suggestion for that)